go.mod

```diff
@@ -2,6 +2,4 @@ module istio.io/gogo-genproto
 
 go 1.12
 
-replace github.com/gogo/protobuf => github.com/istio/gogo-protobuf v1.2.2-0.20190726125433-4c9abdb3090c
-
-require github.com/gogo/protobuf v0.0.0-00010101000000-000000000000
+require github.com/gogo/protobuf v1.2.2-0.20190730201129-28a6bbf47e48
```

Comment:
Wonder how v1.2.2 is picked up, as the latest release is v1.2.1 for `github.com/gogo/protobuf`: https://github.com/gogo/protobuf/releases. 

It's breaking us -- we depends on `envoyproxy/go-control-plane` which depends on this repo, but we use the latest tagged release for `github.com/gogo/protobuf` (i.e. v1.2.1) instead of `master` which apparently has incompatible changes.

Reply:
I'm sorry this is breaking you. Please see this related PR for more details: https://github.com/istio/gogo-genproto/pull/12